server/apps/main/helpers.py

```diff
@@ -364,3 +368,104 @@ def update_public_experience_db(data, uuid, ohmember, editing_user, **change_inf
 
     else:
         delete_PE(uuid, ohmember)
+
+def get_elided_page_range(current, total, on_each_side, on_ends):
```

Comment:
I guess we needed to add this ourselves as our Django version is so old that we don't have access to the now-build-in version of it? 🙈  

If that's the case, maybe we should generally try to see if we can upgrade to a more modern version of Django? If i read the poetry file correct we're currently 2 major releases behind?

Reply:
Yes, exaxctly right. I've rebased these changes on top of your Django upgrade (many thanks for doing that), and as a result have now been able to switch the code to use Django's version of the method instead.